packages/jest-cli/src/init/index.ts

```diff
@@ -11,6 +11,7 @@ import chalk = require('chalk');
 import prompts = require('prompts');
 import {constants} from 'jest-config';
 import {tryRealpath} from 'jest-util';
+import {JEST_CONFIG_EXT_TS} from 'jest-config/build/constants';
```

Comment:
```suggestion
import {JEST_CONFIG_EXT_TS} from 'jest-config';
```

or however you access it - just don't reach into `build` 🙂

Reply:
Nah, it deconstructed from `constants` a bit further down (line 27 ATM)

Reply:
Oh, the IDE decided for me on this one 😄